WindowsServerDocs/get-started-19/servicing-channels-19.md

```diff
@@ -55,6 +59,10 @@ The following table summarizes the key differences between the channels:
 | Who can use | All customers through all channels | Software Assurance and cloud customers only |
 | Installation options | Server Core and Server with Desktop Experience | Server Core for container host and image and Nano Server container image |
 
+> [!IMPORTANT]
+> Please understand that the set of roles and features in Windows Server SAC, only available as Server Core installation option, differs from Windows Server LTSC, installed with Server Core installation option.
+> For example you cannot use Windows Server SAC as foundation for services like S2D (Storage Spaces Direct).
```

Comment:
capitalization/punctuation: The sentence needs to be separated from "For example", coherent with codestyle and readability.
```suggestion
> For example: You cannot use Windows Server SAC as a foundation for services like S2D (Storage Spaces Direct).
```